docs/src/views/AutoFocusZone.tsx

```diff
@@ -16,25 +16,30 @@ export default () => (
     </ul>
     <Header as="h2">Overview</Header>
     <p>
-      {code('AutoFocusZone')} can wrap a component and is used to grab focus and put it to an inner
-      element when the component mounts. For example, when it is needed to focus an inner element in
-      the Popup when it mounts.
+      {code('AutoFocusZone')} can be integrated into {code('Popup')} component and is used to grab
+      focus and put it to an inner element when the component mounts. This can be achieved by
+      specifying {code('autoFocus')} prop on the {code('Popup')}.
     </p>
     <p>
       If you need both - grabbing the focus and trap the focus in the component - use{' '}
-      <Link to="focus-trap-zone">FocusTrapZone</Link>.
+      {code('trapFocus')} prop for {code('Popup')}
+      to integrate <Link to="focus-trap-zone">FocusTrapZone</Link>.
     </p>
     <Header as="h2">Usage</Header>
     <p>
-      In Stardust, {code('AutoFocusZone')} is applied through accessibility behavior, as for{' '}
-      {code('FocusZone')} and
-      {code('FocusTrapZone')}. To enable auto focus for a component, in the behavior set prop{' '}
-      {code('autoFocus')} to {code('true')}
-      with default settings or set an object with desired values for auto focus zone props.{' '}
-      <Link to="accessibility-behaviors">Read more about Accessibility Behaviors.</Link>
+      In {code('Popup')}, set {code('autoFocus')} to {code('true')} with default settings or set an
+      object with desired values for auto focus zone props.
     </p>
+    <p>By default, focus will be set to the first tabbable element in the Popup:</p>
+    <CodeSnippet
+      label="PopupExample.jsx"
+      value={`
+        const Popup = () => (
+          <Popup autoFocus />
+        )`}
```

Comment:
nit: can be one liner:

```jsx
value='const AutoPopup = () => <Popup autoFocus />'
```

**!**: At least name should be changed to avoid collisions with component